docs/sphinxext/docscrape.py

```diff
@@ -0,0 +1,497 @@
+"""Extract reference documentation from the NumPy source tree.
+
+"""
+
+import inspect
+import textwrap
+import re
+import pydoc
+from StringIO import StringIO
+from warnings import warn
+4
```

Comment:
What is the `4` here? This cannot possibly work :thinking:, do we even use it?

Reply:
I don't know. I didn't create this file :)

Reply:
Who did then :)? Can we remove it?
I would rather not commit broken stuff into the repo (not that this one is public and thus I want it to be as clean as possible).